doc/README.md

```diff
@@ -41,6 +41,8 @@
 
 [re-frame usage (recorded meeting)](https://drive.google.com/file/d/1qv_E0CEGzQpu_zGXD0gCTU5EvhC2k8Jy/view)
 
+[status app functionality demo](https://drive.google.com/file/u/0/d/1PmwsMLTLDNNIdv5_6wvMOwoj2PfT50c6/view?usp=drive_web&pli=1)
```

Comment:
Just a nitpick) I'd remove the url parameters from the link `?usp=drive_web&pli=1` to make it clearer)

Reply:
Thank you, I did not noticed that